### PR TITLE
Update zhCN.lua

### PR DESCRIPTION
**Pull request by EKE00372**
Changes proposed in this pull request:

locazilation are gone. make them back.

### CODE REVIEW

File: locale/zhCN.lua [hunk 1]
```diff
@@ -1,26 +1,26 @@
 local L = select(2, ...)('zhCN')
 
 -- Withered scenario, pulled from LibBabble-SubZone-3.0
-L['The Collapse'] = '大陷坑'
```
Comment (p3lim) [inline locale/zhCN.lua]: This one needs to be 100% identical to the Suramar withered scenario zone, which I've pulled from LibBabble-SubZone and should be correct already.

Comment (EKE00372) [reply locale/zhCN.lua]: select zhcn locale client to check last year when translating but ok dont change that until check again

Comment (p3lim) [reply locale/zhCN.lua]: If you can, please provide a screenshot to confirm either way what the name actually says.

Comment (EKE00372) [reply locale/zhCN.lua]: well i cant take screenshot at once...would be later

Comment (EKE00372) [reply locale/zhCN.lua]: in fact i suggest use mapID/instancetype to disable automatic O.o

Comment (p3lim) [reply locale/zhCN.lua]: I was not able to do that before, but I will check again with the new map system, thanks for reminding me.

**Pull request merged**